TestCases/grad_smooth/naca0012/inv_NACA0012_gradsmooth.cfg

```
@@ -0,0 +1,170 @@
+CFL_NUMBER= 25.0
+CFL_ADAPT= NO
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
+OBJECTIVE_FUNCTION= DRAG
+OBJECTIVE_WEIGHT = 1.0
+
+% ----------- SLOPE LIMITER AND DISSIPATION SENSOR DEFINITION -----------------%
+
+MUSCL_FLOW= YES
+SLOPE_LIMITER_FLOW= VENKATAKRISHNAN
+VENKAT_LIMITER_COEFF= 0.03
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+
+CONV_NUM_METHOD_FLOW= JST
+TIME_DISCRE_FLOW= EULER_IMPLICIT
+
+% ----------------------- SOBOLEV GRADIENT SMOOTHING OPTIONS ----------------------%
+%
+% Activate the gradient smoothing solver for the discrete adjoint driver (NO, YES)
+% this will run the Sobolev gradient smoothing solver as an additional postprocessing step
+% to the derivative projections in SU2_DOT_AD.
+% The method requires the AD mode to be active.
+SMOOTH_GRADIENT= YES
+%
+% Mode how the Sobolev method is applied to the sensitivities.
+% - NO_MODUS              (Default option if none is choosen, does nothing.)
+% - PARAM_LEVEL_COMPLETE  (Apply Sobolev smoothing to the projected gradient, w.r.t. the design values.)
+% - MESH_LEVEL            (Apply Sobolev smoothing to the mesh sensitivities, either in the volume or on the surface.)
+% - DEBUG                 (Flag to test the finite elements asembly of the Laplace-Beltrami operator.)
+% - ONLY_GRADIENT         (Use the smoothing solver to calculate the unaltered gradient.)
+SOBOLEV_MODE= PARAM_LEVEL_COMPLETE
+%
+% Scaling factor for the identity part of the Laplace-Beltrami operator
+SMOOTHING_EPSILON1= 1.0
+%
+% Scaling factor for the Laplace part of the Laplace-Beltrami operator
+SMOOTHING_EPSILON2= 0.625
+%
+% Switch for applying the Sobolev smoothing to the design surface mesh, i.e., instead of the whole volume mesh (NO, YES).
+% Only works with SOBOLEV_MODE=MESH_LEVEL.
+SMOOTH_ON_SURFACE= YES
+%
+% Switch for applying the solver to each space dimension of the meshsensitivity seperately.
+% Only works with SOBOLEV_MODE=MESH_LEVEL.
+SEPARATE_DIMENSIONS= NO
+%
+% Use Dirichlet boundary conditions when working on the design surface.
+% Calculates the edge nodes of the design surface and sets the meshsensitivity there =0.
+% Only used for SMOOTH_ON_SURFACE= YES .
+DIRICHLET_SURFACE_BOUNDARY= NO
+%
+% For smoothing meshsensitivities in the volume mesh, the Laplace-Beltrami operator is applied with 0 Neumann boundary by default.
+% Dirichlet boundary conditions can be used by adding boundary markes to MARKER_SOBOLEVBC. The meshsensitivities will be set to 0,
+% i.e. zero Dirichlet boundaries for this marker.
+MARKER_SOBOLEVBC= ( airfoil )
+%
+% Output filename for the assembled Sobolev smoothing system matrix.
+% Writes the projected Laplace-Beltrami operator into a file for external optimization.
+% Requires SOBOLEV_MODE=PARAM_LEVEL_COMPLETE.
+HESS_OBJFUNC_FILENAME= of_hess.dat
+%
+% Linear solver for the finite elements system on the mesh level,
+% supports the same solver options as LINEAR_SOLVER, e.g. (FGMRES, RESTARTED_FGMRES, BCGSTAB),
```

No need to repeat option lists in that case.
```suggestion
% supports the same solver options as LINEAR_SOLVER,
```